markdown-it-mathjax.js

```diff
@@ -63,7 +63,17 @@
         return false
       }
     }
+
+    // find endMarkerPos
     var endMarkerPos = state.src.indexOf(endMarker, startMathPos)
+    for (var i = 0; i < state.src.length; i++) {
+      var marker = state.src[i]
+      var afterMarker = state.src[ i + 1 ]
+      if (marker === endMarker && !!/[^\d]/.test(afterMarker) && (i + 1) > startMathPos) {
```

Comment:
I'd assume the `!!` is not necessary since `test` already returns `true` / `false`? 

Although, honestly, who cares. Just thought I'd mention it.

Reply:
Oh right of course! I thought I needed to coerce it but you're right.  It works with null and even undefined.  Thanks I'll update.